src/main/java/wooteco/subway/web/member/exception/InvalidPasswordException.java

```diff
@@ -0,0 +1,13 @@
+package wooteco.subway.web.member.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.BAD_REQUEST)
+public class InvalidPasswordException extends IllegalArgumentException{
```

Comment:
```suggestion
public class InvalidPasswordException extends IllegalArgumentException {
```

IntelliJ Auto Formatting 기능을 사용하는 습관을 들여보아요.

The auto-formatting shortcut in IntelliJ

* For windows Ctrl + Alt + L.
* For ubuntu Ctrl + Alt + windows + L.
* For Mac Command + Option + L.

Reply:
윽 죄송합니다...! 신경썼어야하는데...! 다음에는 더 신경써서 이런 일 없도록 하겠습니다 ㅠ 거듭 죄송합니다 ㅠㅜ